server/lib/template/helpers/dates.js

```diff
@@ -16,9 +16,10 @@ const TIMESTAMP_FMT = 'DD/MM/YYYY HH:mm:ss';
  * @param {Date} value - the date value to be transformed
  * @returns {String} - the formatted string for insertion into templates
  */
-function date(value) {
+function date(value, dateFormat) {
+  dateFormat = dateFormat.name === 'date' ? DATE_FMT : dateFormat;
```

Comment:
Could this be 
```js
let fmt = dateFormat || DATE_FMT;
let input = moment(value):
return input.isValid() ? input.format(fm) : '';
```
or would that not work?

I like the idea of passing in date parameters, but this code is a bit confusing.